addons/stock/models/product.py

```diff
@@ -916,7 +916,10 @@ def copy(self, default=None):
 
     # Be aware that the exact same function exists in product.product
     def action_open_quants(self):
-        return self.product_variant_ids.filtered(lambda p: p.active or p.qty_available != 0).action_open_quants()
+        if (self.env.context.get('default_product_id')):
+            return self.env['product.product'].browse(self.env.context.get('default_product_id')).action_open_quants()
```

Comment:
```suggestion
        if self.env.context.get('default_product_id'):
            return self.env['product.product'].browse(self.env.context['default_product_id']).action_open_quants()
```
Little nitpick and we're good to go :slightly_smiling_face:

Reply:
This makes sense, I have updated the commit to reflect this. Thanks!